src/app/list-task/list-task.component.ts

```diff
@@ -0,0 +1,45 @@
+import { Component, Input, OnInit, Output, EventEmitter, ViewChild, ElementRef, Renderer2 } from '@angular/core';
+import { ChildActivationEnd } from '@angular/router';
+import { Tarefa } from '../todo-list/tarefa.interface';
+
+
+
+@Component({
+  selector: 'app-list-task',
+  templateUrl: './list-task.component.html',
+  styleUrls: ['./list-task.component.scss']
+})
+export class ListTaskComponent implements OnInit {
+
+  @Input() tarefas: Tarefa[] = [];
+  @Output() aoRemover = new EventEmitter<any>();
+
+  @ViewChild("tarefaDesc") tarefaDesc: HTMLElement;
+
+  index = 0;
+  editField: string;
+
+  constructor(private renderer: Renderer2) { }
+
+  changeValue(HTMLElement) {
+    HTMLElement.focus();
+  }
+
+  deletar(tarefa: Tarefa) {
+    this.aoRemover.emit(this.tarefas.splice( this.tarefas.indexOf(tarefa), 1 ));
```

Comment:
no filho: `this.aoRemover.emit( ( ) => this.tarefas.splice(......) )`
no html do pai: `<app-list-task (deletar)="deletar($event)" />`
no ts do pai: 
```
deletar( func: Function ) {
  func()
}
```